recipe/meta.yaml

```diff
@@ -64,16 +74,140 @@ requirements:
     - libpng
     - libtiff
     - openjpeg
+    - qt  # [not (ppc64le or arm64)]
     - zlib
-  run:
-    - {{ pin_compatible('boost-cpp') }}
-    - poppler-data
+    - nss  # [not win]
 
-test:
-  commands:
-    - pdfinfo -listenc
-    - pdfunite --help
-    - pdftocairo --help
+outputs:
+  - name: poppler
+    files:
+      - {{ prefix }}bin/pdfattach
+      - {{ prefix }}bin/pdfdetach
+      - {{ prefix }}bin/pdffonts
+      - {{ prefix }}bin/pdfimages
+      - {{ prefix }}bin/pdfinfo
+      - {{ prefix }}bin/pdfseparate
+      - {{ prefix }}bin/pdftocairo
+      - {{ prefix }}bin/pdftohtml
+      - {{ prefix }}bin/pdftoppm
+      - {{ prefix }}bin/pdftops
+      - {{ prefix }}bin/pdftotext
+      - {{ prefix }}bin/pdfunite
+      - {{ prefix }}include/poppler/*.h
+      - {{ prefix }}include/poppler/cpp
+      - {{ prefix }}include/poppler/fofi
+      - {{ prefix }}include/poppler/glib
+      - {{ prefix }}include/poppler/goo
+      - {{ prefix }}include/poppler/splash
+      - {{ prefix }}lib/girepository-1.0/Poppler-*.typelib
+      - {{ prefix }}lib/libpoppler.*
+      - {{ prefix }}lib/libpoppler-cpp.*
+      - {{ prefix }}lib/libpoppler-glib.*
+      - {{ prefix }}lib/pkgconfig/poppler.pc
+      - {{ prefix }}lib/pkgconfig/poppler-cpp.pc
+      - {{ prefix }}lib/pkgconfig/poppler-glib.pc
+      - {{ prefix }}share/gir-1.0/Poppler-*.gir
+      - {{ prefix }}share/man/man1/pdfattach.1
+      - {{ prefix }}share/man/man1/pdfdetach.1
+      - {{ prefix }}share/man/man1/pdffonts.1
+      - {{ prefix }}share/man/man1/pdfimages.1
+      - {{ prefix }}share/man/man1/pdfinfo.1
+      - {{ prefix }}share/man/man1/pdfseparate.1
+      - {{ prefix }}share/man/man1/pdftocairo.1
+      - {{ prefix }}share/man/man1/pdftohtml.1
+      - {{ prefix }}share/man/man1/pdftoppm.1
+      - {{ prefix }}share/man/man1/pdftops.1
+      - {{ prefix }}share/man/man1/pdftotext.1
+      - {{ prefix }}share/man/man1/pdfunite.1
+      - {{ prefix }}Library/bin/pdfattach.exe
```

Comment:
Note that at the top of this file, `{{ prefix }}` is defined to include `Library/` on Windows. So you should be able to get rid of most of these; I think there is a variable `EXEEXT` or something that expands to `.exe` on Windows, for the files in `bin`?

Reply:
Oh I did miss that, thanks! I did see some error earlier that prompted me to add the `Library` prefix by hand, but I'll go back and clean it up.